### PR TITLE
feat: add a proper readme

### PR DESCRIPTION
**Pull request by smuu**
Resolves https://github.com/celestiaorg/knuu/issues/75, resolves https://github.com/celestiaorg/knuu/issues/81, resolves https://github.com/celestiaorg/knuu/issues/12

## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,201 @@
...
+Some of the features of knuu are:
+
+- Initialize an Instance from a Container/Docker image
+- Configure startup commands
+- Configure Networking
+    - What ports to expose
+    - Disable networking to simulate network outages
+- Configure Storage
+- Execute Commands
+- Configure resources
```
Comment (Bidon15) [inline README.md:26]:
```suggestion
- Configure HW resources
```

File: README.md [hunk 2]
```diff
@@ -1,9 +1,201 @@
...
+- Create a pool of Instances and control them as a group
+- And more to come...
```
Comment (Bidon15) [inline README.md:28]: I would point to a global pinned issue to direct people somewhere more tangible

File: README.md [hunk 3]
```diff
@@ -1,9 +1,201 @@
...
+
+> If you have feedback on the framework, want to report a bug, or suggest an improvement, please create an issue.
```
Comment (Bidon15) [inline README.md:30]: Permalink to issue

File: README.md [hunk 4]
```diff
@@ -1,9 +1,201 @@
...
+This section will guide you on how to set up and run **knuu**.
+
+### Prerequisites
+
+1. **Docker**: Knuu requires Docker to run. You can install Docker by following the instructions [here](https://docs.docker.com/get-docker/).
+
+2. **Kubernetes cluster**: Set up access to a Kubernetes cluster using a kubeconfig.
+
+3. **'test' Namespace**: Create a namespace called 'test' in your Kubernetes cluster.
+   > **Note:** The used namespace can be changed by setting the `KNUU_NAMESPACE` environment variable.
```
Comment (Bidon15) [inline README.md:43]: [non-blocking][proposal]: it won't hurt to provide more information on how to setup a cluster

**Review by Bidon15 — COMMENTED** [review]
Some nits - otherwise, approved in next commit 👍

File: README.md (hunk 2 above)
Comment (smuu) [reply README.md:28]: What do you mean by that?

Comment (Bidon15) [reply README.md:28]: You can create a high-level issue in knuu repo and pin it 
Then use the link from it and provide it here

**Review by jrmanes — APPROVED** [review]
Nice work! 🙌

**Pull request merged**